privacyidea/static/components/token/views/token.details.html

```diff
@@ -10,13 +10,24 @@ <h3 name="Token Details"><translate>Token details for {{ tokenSerial }}
             <td translate>Type</td>
             <td>{{ token.tokentype }}</td>
             <td>
+                <span>
                 <button class="btn btn-danger" ng-click="deleteTokenAsk()"
                         id="deleteButton"
                         ng-disabled="!checkRight('delete')"
                         ng-hide="!checkRight('delete') && hide_buttons">
                     <span class="glyphicon glyphicon-trash"></span>
                     <span translate>Delete</span>
                 </button>
+                </span>
+                <span>
+                <button class="btn btn-danger" ng-click="regenerateTokenAsk(token)"
+                        id="rolloverButton"
+                        ng-disabled="!regenerateTokenAllowed(token)"
+                        ng-hide="!regenerateTokenAllowed(token) && hide_buttons">
```

Comment:
I would show the button only if the token regenerate is allowed. Otherwise we have the disabled button on every token type (even the ones which do not support rollover anyway) which might confuse people (I know about the 'hide_buttons' but still...).

Reply:
done in https://github.com/privacyidea/privacyidea/commit/90b60853d1020aa950c45f486257760931a1b290